golismero/api/net/web_utils.py

```diff
@@ -126,7 +126,8 @@ def data_from_http_response(response):
 
     # If we have no data, return None.
     if not response.data:
-        return None
+        from ..data.information.html import HTML
```

Comment:
I think there should be an if statement there to prevent calling add_resource when data is None. I prefer that to adding empty objects to the database.